### ISSUE TITLE
Update the CSR and API to support the latest published bluebook available from the DVB website.

### ISSUE THREAD

**Issue by juhajoki** [DVBProject/DVB-I-Reference-CSR#14]
Update the CSR and query API to be compliant with the latest DVB-I bluebook:

[DVB BlueBook A177r4 (Interim draft TS 103 770 V.1.2.1) ](https://dvb.org/wp-content/uploads/2022/01/A177r4_Service-Discovery-and-Programme-Metadata-for-DVB-I_Interim-Draft_TS-103-770-v121_September-2022.pdf)

**Comment by paulhiggs** [comment]
While A177r4 is a draft of TS 103 770 v1. 2.1, it will be A177r5 (expected to be complete in mid-2023) that is anticipated to become the final v1.2.1 standard.

**Comment by juhajoki** [comment]
good point, we will incorporate changes from r5 as well when it's available/stable.

**Comment by juhajoki** [comment]
now returning r6 compatible list. raise new tickets if new features needed / problems encountered.
